### DISCUSSION TITLE
Estimated release of v1.0

### DISCUSSION THREAD

**Discussion by htunnicliff** [vercel/swr#1356] [General] [answered]
Is there an established timeline for the release of version 1.0? 

My team is running into a bug that appears to have been resolved in one of the beta RCs, but it has not been published yet. It would be great to know if the release is soon (we will endure the bug) or if the release is not coming out anytime soon (in that case, we will work on migrating to a beta RC).

The bug in reference is #1345.

**Comment by shuding** [comment, marked as answer]
We plan to release 1.0 next week.
